pipeline.py

```diff
@@ -432,6 +453,20 @@ def ilab_pipeline(
     model_pvc_delete_task = DeletePVC(pvc_name=model_pvc_task.output)
     model_pvc_delete_task.after(final_eval_task)
 
+    generate_metrics_report_task = generate_metrics_report_op()
+    generate_metrics_report_task.after(output_mt_bench_task, final_eval_task)
```

Comment:
I think this depends on `final_eval_task` only, doesn't it?

Reply:
yes it does 😄